src/ast/ast_validations.c

```diff
@@ -748,11 +750,23 @@ static AST_Validation _Validate_UNWIND_Clauses(const AST *ast, char **reason) {
 		const cypher_astnode_t *expression = cypher_ast_unwind_get_expression(unwind_clauses[i]);
 		cypher_astnode_type_t type = cypher_astnode_type(expression);
 		// Ensure that the UNWIND expression is a collection (aka list/array)
-		// if(type != CYPHER_AST_COLLECTION) {
-		// 	asprintf(reason, "UNWIND expects a list argument; encountered ''%s'", cypher_astnode_typestr(type));
-		// 	res = AST_INVALID;
-		// 	goto cleanup;
-		// }
+		if(type == CYPHER_AST_APPLY_OPERATOR) {
+			const cypher_astnode_t *funcNode =  cypher_ast_apply_operator_get_func_name(expression);
+			const char *funcName = cypher_ast_function_name_get_value(funcNode);
+			size_t lower_func_name_len = 32;
+			char lowerFuncName[lower_func_name_len];
+			_toLower(funcName, lowerFuncName, &lower_func_name_len);
+			if(strcmp(lowerFuncName, "range")) {
```

Comment:
use `strcasecmp` from strings.h to perform a case insensitive comparison,
this make common.h obsolete.

Reply:
done